recipe/meta.yaml

```diff
@@ -138,10 +139,8 @@ outputs:
       run:
         - python
         - cython
-        - {{ pin_subpackage('hyperspy-base', exact=True) }}
```

Comment:
Remove this dependency to avoid pinning hyperspy dependencies to the release version one (can be annoying when using the development version) and enable the following to install hyperspy in development mode:
```
conda install hyperspy-base -c conda-forge --only-deps 
conda install hyperspy-dev -c conda-forge
pip install -e . --no-deps
```